rts/at/sra/README.md

```diff
@@ -0,0 +1,68 @@
+OVERVIEW
+--------
+An example of how to use REST-AT service.
+
+USAGE
+--------------------
+TEST SRA PARTICIPANTS
+
+Start a REST-AT coordinator on port 8080
+
+    cd <narayana-repo>/jboss-as/build/target/wildfly-28.0.0.Final
+    cp docs/examples/configs/standalone-rts.xml standalone/configuration
+    ./bin/standalone.sh -c standalone-rts.xml
+
+VERIFY THAT IT IS RUNNING, eg:
+
+    curl http://localhost:8080/rest-at-coordinator/tx/transaction-manager
+
+BUILD  AND RUN THE DEMO APPLICATION on port 8081
+
+    cd <narayana-repo>/rts/sra
+    mvn clean package -Dquarkus.package.type=uber-jar
+    java $JAVA_OPTS -Dquarkus.http.port=8081 -jar target/sra-participant-runner.jar &
+    
+    OR
+    mvn quarkus:dev
```

Comment:
Does this command start the service on port 8081?

Reply:
> Does this command start the service on port 8081?

Yes, If used the second command give in the OR it will pickup from properties and default port. I gave two option.

Reply:
Okay.

Why do you need to specify `-Dquarkus.http.port=8081` when running `java $JAVA_OPTS -Dquarkus.http.port=8081 -jar target/sra-participant-runner.jar`?

Reply:
This is updated, so resolving the conversation.